pkg/config/utils/endpoints.go

```diff
@@ -111,6 +111,25 @@ func GetMainEndpoint(c pkgconfigmodel.Reader, prefix string, ddURLKey string) st
 	return prefix + pkgconfigsetup.DefaultSite
 }
 
+// GetHAEndpoint returns the HA DD URL defined in the config, based on `ha.site` and the prefix, or ddURLKey
+func GetHAEndpoint(c pkgconfigmodel.Reader, prefix string, ddHaUrlKey string) string {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏Let's make sure with HARM agent folks that this isn't defined elsewhere

Reply:
Good call! I started a 🧵 https://dd.slack.com/archives/C06G1JU6W3T/p1706650468449579